.concourse/kubecf.yaml

```diff
@@ -1,5 +1,8 @@
 ---
 
+workertags:
```

Comment:
Should we add a comment on how this is supposed to be used? E.g.
`Specify a worker tag if you want your whole pipeline to run on specific (tagged) workers. This can be used to prevent resource sharing between your pipeline and the production one.`

Reply:
done!